tools/templates/lint-singleuser-extraContainers.yaml

```diff
@@ -0,0 +1,15 @@
+# hint for config singleuser extraContainers
+# The same with hub.extraContainers and singleuser.initContainers
+# Kubespawner support config extraContainers:
+# https://github.com/jupyterhub/kubespawner/blob/master/kubespawner/spawner.py#L845
+# And the containers spec can follows kubernetes container-v1-core.
+# https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.11/#container-v1-core.
+#
+# For now, the extraContainers' image will be prepull in daemonset. It can also be removed
+# from prepull daemonset if necessary
+
+singleuser:
+  extraContainers:
+  - name: mock-extra-container-name
+    image: mock-extra-container-image-name
+    imagePullPolicy: IfNotPresent
```

Comment:
This is exactly the kind of stub content I wanted, but could you add it to [`lint-and-validate-values.yaml`](https://github.com/jupyterhub/zero-to-jupyterhub-k8s/blob/master/tools/templates/lint-and-validate-values.yaml) instead? This new file wont be automatically tested by the CI or the `lint-and-validate.py` script, only the one named [`lint-and-validate-values.yaml`](https://github.com/jupyterhub/zero-to-jupyterhub-k8s/blob/master/tools/templates/lint-and-validate-values.yaml) is utilized.